ENT-329: Success Message for Account Activation for logged out users.

saleem-latif opened the pull request:
@brittneyexline , @douglashall , @asadiqbal08 , @zubair-arbi 

This PR contains updates for account activation messages for logged out users.

__Jira Ticket__: [ENT-329](https://openedx.atlassian.net/browse/ENT-329)

__Description:__
If a learner does not have an activated account, and is signed out. When the learner clicks the link to activate their account from their email, the system changes their account flag to active. The learner still needs to sign in. This story is to redirect the learner from that Successful Activation page with the giant green fake not-a-button, to the actual sign in page with an Account Activation success message displaying.

__Acceptance Criteria__
1. When a signed-out learner clicks the account activation link from their email, the system will activate their account and display the Sign In page
2. On the sign in page, display the account activation success message

douglashall commented:
jenkins run python

douglashall commented:
It looks like the styling of the alerts for both logged-in and logged-out users is a bit off in the WL themes. Can you see if there is something we can do to fix this either in the edx-themes repo or edx-platform?

<img width="1034" alt="screen shot 2017-05-19 at 4 14 34 pm" src="https://cloud.githubusercontent.com/assets/224634/26265317/b5dd9738-3cae-11e7-950c-bb073d9efe3c.png">
<img width="967" alt="screen shot 2017-05-19 at 1 45 11 pm" src="https://cloud.githubusercontent.com/assets/224634/26265318/b5deabb4-3cae-11e7-9f00-488c5830cc8f.png">

saleem-latif commented:
@douglashall I have added tests for account activation message and also fixed the problem with white label sites, please take a look.
<img width="1273" alt="screen shot 2017-05-22 at 4 41 17 pm" src="https://cloud.githubusercontent.com/assets/7114956/26307391/b00f64c4-3f0d-11e7-852c-af2278f40075.png">
<img width="1280" alt="screen shot 2017-05-22 at 4 41 55 pm" src="https://cloud.githubusercontent.com/assets/7114956/26307393/b0eebe80-3f0d-11e7-9df3-d0611b885baa.png">

saleem-latif commented:
jenkins run bokchoy

saleem-latif commented:
jenkins run bokchoy

saleem-latif commented:
jenkins run bokchoy
